packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/reducer.js

```diff
@@ -280,68 +218,51 @@ const reducer = (state, action) =>
          */
         const data = cloneDeep(initialValues);
 
-        /**
-         * relationalFieldPaths won't be an array which is what we're expecting
-         * Therefore we reset these bits of state to the correct data type
-         * which is an array. Hence why we replace those fields.
-         */
-        const mergeDataWithPreparedRelations = relationalFieldPaths
```

Comment:
QA will and the tests will probably cover that automatically: where do you handle relations now, which are not part of components? From the code this is not immediately clear to me.

Reply:
`findAllRelationsAndReplaceWithEmptyArray` im not sure what line this is on.